velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -819,6 +882,20 @@ BufferPtr VectorFuzzer::fuzzIndices(
   return indices;
 }
 
+BufferPtr VectorFuzzer::fuzzInMap(vector_size_t size) {
+  VELOX_CHECK_GE(size, 0);
+  auto inMap = AlignedBuffer::allocate<bool>(size, pool_, 0);
+  auto mutableInMap = inMap->asMutable<uint64_t>();
+
+  for (auto row = 0; row < size; ++row) {
+    if (coinToss(.25)) {
```

Comment:
make this configurable as well; it could be useful to change this for test purposes.